rust_src/src/casefiddle.rs

```diff
@@ -16,7 +16,7 @@ use threads::ThreadState;
 use obarray::intern;
 use symbols::symbol_value;
 
```

Comment:
`casify_word` is not a `lisp_fn`. Leave the useless return.

Reply:
Ok, nevermind. It looks like the return was only added to satisfy the DEFUN return requirement.